### PR TITLE
Adds a guillotine

### PR DESCRIPTION
**Pull request by Kmc2000**
Refer to issue: Lizards exist

Adds a guillotine, craftable with wood, a butcher's cleaver and wire. You can use it to behead normal people or to de-tail dirty lizard scum.

https://i.gyazo.com/38a7c29e7d2526de68a4851b97746779.mp4

#### Changelog

:cl: Kmc2000
rscadd: Added a guillotine, build yours today!
/:cl:

### CODE REVIEW

**Comment by AsV9** [comment]
good pr

**Comment by Nickvr628** [comment]
Does de-tailing the lizards do damage to them? Or is it a "humane" removal?

**Comment by Ratvar** [comment]
Will it behead a lizard without a tail? Is it instant? Banana guillotine revs confirmed
Where is bleeding effect and screams?
Can I cut watermellons with it?

File: code/game/objects/structures/guillotine.dm [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+/obj/structure/guillotine/relaymove(mob/living/restrained, direction)
+	TryEscape(restrained, restrained)
+
+
+/obj/structure/guillotine/attackby(obj/item/I, mob/living/user, proximity)
+	if(!upgraded)
+		if(istype(I, /obj/item/stack/cable_coil))
+			to_chat(user, "You haphazardly wind [I] around [src], increasing its revolutions per minute as well as the time that it will take to free someone from it.")
+			desc += "-it seems to have some cables woven around the stocks. It'll take extra time to escape from it."
+			escapeTime += 30
```
Comment (Time-Green) [inline code/game/objects/structures/guillotine.dm:43]: Probably wanna set upgraded to TRUE so people cant infinitely add cable coil to it

Comment (Kmc2000) [reply code/game/objects/structures/guillotine.dm:43]: Good point lol

**Comment by Kmc2000** [comment]
De-tailing also chops their legs off because physics.

Yeah it works in TAIL mode or HEAD mode exclusively.
Bleeding happens when your limbs are cut off. Screams possible.

No though I may add that in the future (coding that would be a headache for now)

File: code/game/objects/structures/guillotine.dm (hunk 1 above)
Comment (Time-Green) [reply code/game/objects/structures/guillotine.dm:43]: Also, at no point does it consume the cable

**Comment by Time-Green** [comment]
Also it wouldnt be hard having it cut melon. There has to be a function for it somewhere...

**Comment by Kmc2000** [comment]
It's more allowing you to put objects on, which would require the use of overlays.

Fuck.

Overlays 

:))

**Comment by Time-Green** [comment]
guilotine.overlays += object
And if you want it gone
Guilotine.overlays.Cut()

**Comment by Amelia0010** [comment]
image
🤔

**Pull request closed without merging**